src/main/java/com/kenyajug/regression/utils/DateTimeUtils.java

```diff
@@ -151,15 +152,13 @@ public static String localDateTimeToUTCTime(LocalDateTime localDateTime) {
         var zonedDateTime = ZonedDateTime.of(localDateTime, ZoneId.of("UTC"));
         return convertUTCZonedDateTimeToString(zonedDateTime);
     }
-    public static LocalDateTime fromTomcatLogTimestamp(String timestamp){
+    public static LocalDateTime fromTomcatLogTimestamp(String timestamp) {
         try {
-            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MMM-yyyy HH:mm:ss.SSS");
-            var temporal = formatter.parse(timestamp);
-            return LocalDateTime.from(temporal);
-        } catch (DateTimeParseException formatException){
-            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSX");
-            var temporal = formatter.parse(timestamp);
-            return LocalDateTime.from(temporal);
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MMM-yyyy HH:mm:ss.SSS", Locale.ENGLISH);
+            return LocalDateTime.parse(timestamp, formatter);
+        } catch (DateTimeParseException formatException) {
+            DateTimeFormatter fallbackFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSX");
+            return LocalDateTime.parse(timestamp, fallbackFormatter);
```

Comment:
How does this affect the issue this PR is meant to address? Logs loading performance